src/components/navbar/index.tsx

```diff
@@ -22,22 +21,21 @@ const NavLogo: AnyStyledComponent = styled(Link)`
   }
 `;
 
-const NavLink: AnyStyledComponent = styled(Link)`
-  color: black;
-  font-size: 24px;
-  text-decoration: none;
-
-  &:hover {
-    color: grey;
-    text-decoration: none;
-  }
-`;
-
 const Navbar: React.FC = (): JSX.Element => {
+  const [, setSelectorDisplay] = useGlobal("selectorDisplay");
+  const [, setTutorialDisplay] = useGlobal("tutorialDisplay");
+
   return (
     <NavWrapper>
-      <NavLogo to="/">Propguard</NavLogo>
-      <NavLink to="/tutorial">Tutorial</NavLink>
+      <NavLogo
+        to="/"
+        onClick={(): void => {
+          setTutorialDisplay("none");
```

Comment:
Look for the comment I made on `src/pages/index.tsx` about the type of the global state for tutorialDisplay. Essentially I would prefer it to be a boolean.